app/views/messages/index.html.erb

```diff
@@ -23,13 +23,31 @@
 
               <% if contact_record.contact_record_type == :outgoing_text_message %>
                 <span class="contact-record__phone_number">
-                  Text to <%= contact_record.client.formatted_phone_number %>
+                  Text to <%= local_phone_number(contact_record.client.phone_number) %>
                 </span>
                 <span class="contact-record__status">
                   <%= contact_record.twilio_status || "sending..." %>
                 </span>
               <% end %>
 
+              <% if contact_record.contact_record_type == :incoming_text_message %>
+                <span class="contact-record__phone_number">
+                  Text from <%= local_phone_number(contact_record.from_phone_number) %>
+                </span>
+              <% end %>
+
+              <% if contact_record.contact_record_type == :outgoing_email %>
+                <span class="contact-record__phone_number">
+                  Email to <%= contact_record.client.email_address %>
```

Comment:
I think we should adjust our outgoing email model so we store the email address we sent the message to, then show that here. I'll likely do that, just logging it.